section1/reflection.md

```diff
@@ -1,19 +1,31 @@
 ## Section 1 Reflection
```

Comment:
Your work in Section 1 looks great. It is consistent - which is key - use of spaces, empty lines, double quotes, etc. which makes it very readable 🙌